articles/po3rin.re

```diff
@@ -1249,11 +1249,14 @@ curl -X DELETE localhost:12380 \
     -d '{"id": "4", "url": "http://127.0.0.1:42379"}'
 //}
 
-これでクラスターの動的な設定変更機能が完成しました。
+これでクラスターの動的な設定変更機能が完成しました。さらなる応用として、etcdはv3.4からRaftに「リーダー、フォロワー、候補者」に次ぐ4つ目の状態である@<b>{learner（学習者）}@<fn>{learner}を実装しています(コンセンサスアルゴリズムの実践を考察した論文@<fn>{raft-plactice}では@<b>{non-vorting member（非投票者）}として導入されている)。これはノードの新規追加時にリーダーのログに追いつくまで投票権の無いメンバーとしてクラスターに参加させるというアイデアです。この方法はログ複製でリーダーが過負荷になるのを防ぎ、クラスター再構成の安全性を高めます。
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Line 1252 sentence length(135) exceeds the maximum sentence length of 100.
Over 35 characters. (ja-technical-writing/sentence-length)